package.json

```diff
@@ -102,7 +102,7 @@
     "hoek": "^5.0.4",
     "human-to-milliseconds": "^1.0.0",
     "interface-datastore": "~0.6.0",
-    "ipfs-api": "ipfs/js-ipfs-api",
+    "ipfs-api": "ipfs/js-ipfs-api#feat/cid-base-option2",
```

Comment:
😆 that's the reason I didn't merge it yet

Reply:
Oh I'm sorry, I thought I had read somewhere that PR was ready to merge, the CI was just failing because something else. I might have read that in another PR and thought it was here

Reply:
You did, no apologies necessary, I deleted that comment after I read your review!